compiler/test/dotty/tools/vulpix/VulpixUnitTests.scala

```diff
@@ -1,107 +1,99 @@
 package dotty.tools
 package vulpix
 
-import scala.language.unsafeNulls
-
-import java.io.{File => JFile}
-import org.junit.Assert._
-import org.junit.{ Test, AfterClass }
-
-import scala.concurrent.duration._
-import scala.util.control.NonFatal
+import org.junit.{Test as test, AfterClass as tearDown}
```

Comment:
I find that confusing. Aliases here makes the code harder to follow: each time someone sees `@test` or `tearDown`, they’ll need to check the import to realize these are actually `JUnit`’s `@Test` and `@AfterClass`.

Reply:
I borrowed that idiom from bishabosha, I think. I just saw it recently and fell in love. However, I am not wedded to it (at present).